pkg/cluster/cluster.go

```diff
@@ -110,6 +110,12 @@ func (c *Cluster) Start(ctx context.Context) (<-chan struct{}, error) {
 		}()
 	}
 
+	go func() {
+		if err := c.managedDB.StoreSnapshotData(ctx); err != nil {
+			logrus.Error(err)
```

Comment:
We are going to just log and forget this error, so I think it should be wrapped with a message providing context and making it unique. Something like (but not necessarily exactly) "Failed to record snapshots for cluster."